src/main/java/net/greghaines/jesque/meta/dao/impl/QueueInfoDAORedisImpl.java

```diff
@@ -225,22 +225,39 @@ private long size(final Jedis jedis, final String queueName) {
     }
 
     /**
-     * Get list of payload from a queue.
+     * Get list of JobPayload from a queue.
      * 
      * @param jedis
      * @param queueName
      * @param jobOffset
      * @param jobCount
      * @return
      */
-    private Collection<String> paylods(final Jedis jedis, final String queueName, final long jobOffset, final long jobCount) {
+    private Collection<JobPayload> getJobPayloads(final Jedis jedis, final String queueName, final long jobOffset, final long jobCount) {
```

Comment:
What if this method just returned `Collection<Job>`? Since you're iterating over the elements here, might as well create the Job instead of an intermediary JobPayload object. Will simplify getQueueInfo() quite a bit, too.